client/wfprev-war/src/main/angular/src/app/components/resizable-panel/resizable-panel.component.html

```diff
@@ -1,19 +1,11 @@
-<div class="resizable-panel" [style.width]="panelWidth">
-  <mat-tab-group [(selectedIndex)]="selectedTabIndex" class="tab-group">
-    <mat-tab label="Projects">
-      <app-projects-list></app-projects-list>
-    </mat-tab>
-    <mat-tab label="Dashboard">
-      Content 2
-    </mat-tab>
-    <mat-tab label="Planning">
-      Content 3
-    </mat-tab>
-  </mat-tab-group>
+<div class="resizable-panel">
+  <div class="scrollable-content">
+    <app-projects-list></app-projects-list>
+  </div>
 
-  <div class="panel-controls">
+  <!-- <div class="panel-controls">
```

Comment:
Can you remove this instead of commenting it out?

Reply:
Sure, we will likely bring this back in T2, but I can remove this for now